Shreya/HTML Basics Remake/form.html

```diff
@@ -0,0 +1,40 @@
+<html>
+    <body>
+        
+        <form>
+                <h1>Register</h1>
+           <p>
+                <label for="first">First Name</label>
+                <input type="text" name="first" placeholder="First Name" required>
+                <label for="last">Last Name</label>
+                <input type="text" name="last" placeholder="Last Name" required>
+           </p>
+            <p>
+                <label for="gender">Gender:</label>
+                <label for="male">Male:</label>
+                <input type="radio" id="male" required>
+                <label for="female">Female:</label>
+                <input type="radio" id="female" required>
+                <label for="others">Others:</label>
+                <input type="radio" id="others" required>
```

Comment:
Radio buttons are not present in one group thus leading to the selection of multiple.